theano/tensor/elemwise.py

```diff
@@ -1714,19 +1705,14 @@ class All(CAReduce):
     Equivalent to `CAReduce(scalar.and\_, axis=axis)`.
 
     """
+    __props__ = ("axis",)
 
     def __init__(self, axis=None):
         CAReduce.__init__(self, scalar.and_, axis)
 
     def _output_dtype(self, idtype):
         return "int8"
 
-    def __str__(self):
```

Comment:
Add this back. This class (All) inherit from CAReduce that implement \__str__. We don't want CARedure \__str__.